pystiche_papers/sanakoyeu_et_al_2018/_data.py

```diff
@@ -247,6 +253,62 @@ def image_transform(impl_params: bool = True, edge_size: int = 768) -> nn.Sequen
     return nn.Sequential(*transforms_)
 
 
+def images() -> DownloadableImageCollection:
+    base_sanakoyeu = "https://raw.githubusercontent.com/CompVis/adaptive-style-transfer/gh-pages/images/experiments/comparison_with_others/"
```

Comment:
I suggest you implement something like

```python
def url(
    id: int,
    file: str,
    base: str = "https://raw.githubusercontent.com/CompVis/adaptive-style-transfer/gh-pages/images/experiments/comparison_with_others/",
) -> str:
    return urljoin(base, f"{id:04d}/{file}")
```

and use that throughout this function. The double `urljoin` feels clumsy.